images-list

```diff
@@ -23,6 +23,7 @@ coredns/coredns rancher/coredns-coredns 1.7.0
 coredns/coredns rancher/coredns-coredns 1.7.1
 coredns/coredns rancher/coredns-coredns 1.8.0
 coredns/coredns rancher/coredns-coredns 1.8.1
+coredns/coredns rancher/coredns-coredns 1.8.3
```

Comment:
Is it possible to move code to rancher/mirrored-coredns-coredns ? If not we can move forward as-is. But want to do that soon.

Reply:
I'm fine to do that. This is due for a dev complete of tomorrow. Can we do this after the next k3s release?